### PR TITLE
{lib}[GCCcore/10.3.0] elfutils v0.185

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/b7d30c27f94ff3564d9fa781ca9a21b5 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/73e9ba509351d3f4b7dafd8fe98bb34f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/6ea500f35c5011d8b122ee177068c964 for a full test report.

File: easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ('bzip2', '1.0.8'),
+    ('libarchive', '3.5.1'),
+    ('XZ', '5.2.5'),
+    ('zstd', '1.4.9'),
+]
+
+configopts = "--disable-debuginfod --disable-libdebuginfod"
+
+sanity_check_paths = {
+    'files': ['bin/eu-elfcmp', 'include/dwarf.h', 'lib/libelf.so'],
```
Comment (branfosj) [inline easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb:35]:
```suggestion
    'files': ['bin/eu-elfcmp', 'include/dwarf.h', 'lib/libelf.%s' % SHLIB_EXT],
```

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4063be223164c7241795b957857cdc4a for a full test report.

File: easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+homepage = 'https://elfutils.org/'
+
+description = """
+ The elfutils project provides libraries and tools for ELF files
+ and DWARF data.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://sourceware.org/elfutils/ftp/0.183/']
```
Comment (branfosj) [inline easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb:15]: This `source_urls` is for a previous version?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb (hunk 2 above)
Comment (Micket) [reply easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb:15]: I have no idea how i managed to download the sources

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/75a3a99eef83d427fd1c2518d7763f56 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/f8956150bf5d67a4d0f288b673de5007 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/15d1dfa2c2412cbc324843307b0f7e1a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/7cb00ae6532911fd6a42c03e3f803086 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**